templates/css/product.css

```diff
@@ -48,45 +61,45 @@ amp-carousel {
 
 .header input[type=submit] {
   position:absolute;
-  top:10px;
-  right:8px;
-  background-color:#eb407a;
+  top:var(--space-1);
+  right:var(--space-1);
+  background-color:var(--color-primary);
   background-position:center;
   background-repeat:no-repeat;
   background-image:-webkit-image-set(url(/img/ic_search_white_1x_web_24dp.png) 1x,url(/img/ic_search_white_2x_web_24dp.png) 2x);
   border:0;
-  height:32px;
+  height:var(--space-4);
   width:36px;
   -webkit-appearance:none;
   border-radius:4px 0
 }
 
 .header input[type=search],.header input[type=text] {
   position:absolute;
-  top:10px;
-  right:24px;
-  background:#ededed;
-  padding:0 8px;
+  top:var(--space-1);
```

Comment:
I was wrong, this needs to be 10px for the input field to be centered vertically